Src/Newtonsoft.Json/Utilities/LinqBridge.cs

```diff
@@ -724,7 +724,7 @@ private static IEnumerable<TSource> ReverseYield<TSource>(IEnumerable<TSource> s
     }
 
     /// <summary>
-    /// Returns an <see cref="Int64"/> that represents the total number 
+    /// Returns a <see cref="Int64"/> that represents the total number
```

Comment:
This is somewhat counterintuitive. At least in Visual Studio's C# code editor, `Int64` will be replaced with the C# keyword `long` in IntelliSense tooltips. "A long…" reads better than "An long…".